parser/src/snapshots/rustpython_parser__string__tests__fstring_parse_self_documenting_base.snap

```diff
@@ -14,7 +14,7 @@ expression: parse_ast
     ),
     Constant(
         ExprConstant {
-            range: 0..10,
+            range: 2..9,
```

Comment:
TODO: Inspect the source for this test. It seems strange that all values have the same range. This constant is especially strange because the value is an empty string.

Reply:
this is an intermediate representation which is later merged with the previous constant. (it represents optional spaces between the `=` and the closing `}`)